cmd/airflow_test.go

```diff
@@ -128,13 +128,13 @@ func (s *AirflowSuite) TestDevInitCommandSoftware() {
 	})
 }
 
-func (s *AirflowSuite) TestNewAirflowInitCmd() {
-	cmd := newAirflowInitCmd()
+func (s *AirflowSuite) TestNewAirflowDevRootCmd() {
+	cmd := newDevRootCmd(nil, nil)
 	s.Nil(cmd.PersistentPreRunE(new(cobra.Command), []string{}))
 }
 
-func (s *AirflowSuite) TestNewAirflowStartCmd() {
-	cmd := newAirflowStartCmd(nil)
+func (s *AirflowSuite) TestNewAirflowInitCmd() {
+	cmd := newAirflowInitCmd()
```

Comment:
Weird looking change here, but I've:
- removed the test for `TestNewAirflowStartRootCmd`, the `PersistentPreRunE` function has been moved to the parent command so this test fails and really isn't necessary any more.
- added the test for the `TestNewAirflowDevRootCmd` since it now holds the shared `PersistentPreRunE` function.
- left the test for `TestNewAirflowInitCmd` in place.